prow.sh

```diff
@@ -52,23 +65,38 @@ configvar () {
     eval echo "\$3:" "$1=\${$1}"
 }
 
-# Takes the minor version of $CSI_PROW_KUBERNETES_VERSION and overrides it to
-# $1 if they are equal minor versions. Ignores versions that begin with
-# "release-".
+# Updates the patch version of CSI_PROW_KUBERNETES_VERSION such that we
+# have a kind image for it. Does nothing if the current value isn't a
+# version number.
 override_k8s_version () {
-    local current_minor_version
-    local override_minor_version
+    # Not a version number?
+    if ! echo "$CSI_PROW_KUBERNETES_VERSION" | grep -q '^[0-9][0-9]*\.[0-9][0-9]*'; then
```

Comment:
valid values also include "release-1.17", etc

Reply:
That's exactly why this check is here: it makes `override_k8s_version` return early because there is nothing to be patched in `release-1.17`. For those versions we'll build the image ourselves instead of using a pre-built one.